silk/tools/wpan_table_parser.py

```diff
@@ -446,7 +446,8 @@ def parse_list(list_string):
     # For each line, skip the first two characters (which are `\t"`) and last character (`"`), then split the string
     # using whitespace as separator. The first entry is the IPv6 address.
     #
-    return [line[2:-1].split()[0] for line in list_string.split("\n")[1:-1]]
+    # return [line[2:-1].split()[0] for line in list_string.split("\n")[1:-1]]
+    return [line[2:-1].split()[0] for line in list_string.split("\n")[1:-2]]
```

Comment:
Please run other test cases for this change.

Reply:
Tested with other relevant tests(slaac tests, inform_parent,parent_reset_child_recovery, child_attach_with_multiple_ip_addr) all are PASS.